dcrpulse/umbrel-app.yml

```
@@ -0,0 +1,51 @@
+manifestVersion: 1
+id: dcrpulse
+name: Decred Pulse
+tagline: Full Decred node with watch-only wallet and database-free block explorer
+category: finance
```

Crypto category would probably fit the best here.
```suggestion
category: crypto
```